src/main/scala/is/hail/methods/LinearMixedRegression.scala

```diff
@@ -355,8 +413,10 @@ object DiagLMM {
           val (_, logdetCdC) = logdet(CdC)
           val (_, logdetCtC) = logdet(CtC)
 
-          -0.5 * (logdetD + logdetCdC - logdetCtC + (n - d) * math.log(sigma2)) + shift
+          val res = -0.5 * (logdetD + logdetCdC - logdetCtC + (n - d) * math.log(sigma2)) + shift
```

Comment:
My bad. Did this to print results when things weren't working right, forgot to put it back.